SpatialGDK/Documentation/known-issues.md

```diff
@@ -15,5 +15,5 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | We don't support the `ReplicateYes` policy on GameplayAbilities. | 2018-10-24 | [UNR-675](https://improbableio.atlassian.net/projects/UNR/issues/UNR-675) | Don't use replicated GameplayAbilities. If they need access to replicated data, store it on the AbilityComponent itself. |
```

Comment:
We don't 
-> 
The GDK doesn't

[More grown-up and consistent with the rest of the docs...]